index.d.ts

```diff
@@ -174,12 +174,12 @@ interface SwiperControlsProps {
     /**
      * Custom prev button component
      */
-    PrevComponent?: React.ComponentClass;
+    PrevComponent?: React.ComponentType<T>;
```

Comment:
Same with these type generics

Reply:
I can't commit those change from the UI, can you?